DebuggableASTInterpreter/DASTContext.class.st

```diff
@@ -522,7 +522,8 @@ DASTContext >> methodOrBlock [
 { #category : #accessing }
 DASTContext >> methodOrBlock: aDASTClosure [
 
-	closure := aDASTClosure
+	closure := aDASTClosure.
+	closure doSemanticAnalysis
```

Comment:
Are you sure we want to put it in the setter? I would have expected to do it for example in the method lookup, when the AST is found or created.

Reply:
I hesitated. It would be better indeed but when do we know that the AST is found or created ?